tests/e2e-pw/test-e2e-pw.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+echo "🎭 Running Playwright e2e tests in default headless mode.";
+
+# When updating the Playwright version in the image tage below, make sure to update the npm `@playwright/test` package.json version as well.
+
+docker run -v "$PWD:/woopayments" -w /woopayments \
+	-e BASE_URL=http://host.docker.internal:8084 \
+	-it --rm --ipc=host \
+	mcr.microsoft.com/playwright:v1.39.0-jammy \
+	npx playwright test --config=tests/e2e-pw/playwright.config.ts "$@"
```

Comment:
Do you think it's better to have the docker config outside the shell script? Like the ones we have for Puppeteer E2E tests?

Reply:
👍 I agree, I've had a crack at this via a `docker-compose.yml` file in 660c077, let me know what you think.

Reply:
I assume this has broken the PW GH action – I'll look into it.

Reply:
Now resolved 🙂